### ISSUE TITLE
explicit negation of service

### ISSUE THREAD

**Issue by cKlee** [dini-ag-kim/service-ontology#5]
@dini-ag-kim/bestandsdaten Under the open world assumption you cannot tell if a service is provided until it's stated explicitly.  It might be useful to state that a service is NOT provided. Therefor a property like service:excludes could be used.

Example:

```
$library service:excludes [
    a dso:InterLoan
] .
```

**Comment by nichtich** [comment]
DAIA includes property `daia:unavailableFor` for this use case, does this fit?

**Comment by cKlee** [comment]
Nope! `daia:unavailableFor` has the domain `frbr:Item`. I want to express the unavailability of a service by an institution (agent, organization, library). In the ISIL-Verzeichnis we have explicit data about an organization and its ability towards library interloan.
